packages/core/contracts/oracle/implementation/Staker.sol

```diff
@@ -1,8 +1,6 @@
 // SPDX-License-Identifier: AGPL-3.0-only
 pragma solidity ^0.8.0;
 
-import "../../common/implementation/Testable.sol";
```

Comment:
This needs to be added to `StakerTest.sol` when removed from here. 

I also removed Testable from Staker in this PR: https://github.com/UMAprotocol/protocol/pull/4095